presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -198,18 +234,22 @@ public void testCoordinatorNotJoining()
                 .hasMessage("No coordinator node available");
     }
 
-    @Test
-    public void testCacheRead()
+    @Test(dataProvider = "readMode")
+    public void testCacheRead(ReadMode readMode)
             throws Exception
     {
-        Path file = getStoragePath("some_file");
+        initializerCachingFileSystem(new RubixConfig().setReadMode(readMode));
+        byte[] randomData = new byte[(int) SMALL_FILE_SIZE.toBytes()];
```

Comment:
nit: migrating to randomData is unrelated. Would be nice to have separate commit with that. (not enforcing)

Reply:
It's kind of related. `getAsyncDownloadedMb` is the only `BookKeeper` metric that is updated after fetch request is completed. It requires at least 1MB, so I needed to update the test.